lib/cache/cache.js

```diff
@@ -509,6 +509,8 @@ class Cache {
         requestObject[kHeaders][kHeadersList] = request.headersList
         requestObject[kHeaders][kGuard] = 'immutable'
         requestObject[kRealm] = request.client
+        requestObject[kSignal] = new AbortController().signal
+        requestObject[kSignal][kRealm] = requestObject[kRealm]
```

Comment:
this doesn't actually do anything? The abort contorller is just thrown away?

Reply:
This is necessary because AbortSignal is not set [here](https://github.com/nodejs/undici/blob/1b1ae6b60a058252dcaa655e9498b5bbad77bbde/lib/fetch/request.js#L363) because the new Request step was skipped in #2610.

The signal property reads kSignal.

https://github.com/nodejs/undici/blob/1b1ae6b60a058252dcaa655e9498b5bbad77bbde/lib/fetch/request.js#L715-L720

Reply:
this is why I didn't bother originally, causes too many issues